modules/gdscript/gdscript_compiler.cpp

```diff
@@ -104,7 +104,8 @@ GDScriptDataType GDScriptCompiler::_gdtype_from_datatype(const GDScriptParser::D
 			if (p_handle_metatype && p_datatype.is_meta_type) {
 				result.kind = GDScriptDataType::NATIVE;
 				result.builtin_type = Variant::OBJECT;
-				result.native_type = GDScriptNativeClass::get_class_static();
+				//result.native_type = GDScriptNativeClass::get_class_static();
+				result.native_type = Object::get_class_static(); // GH-82255.
```

Comment:
There's no detailed explanation on #82255 of why this caused a crash and what's the effect of doing this change here. So might be worth discussing in this PR / maybe clarifying as a comment.